backend/.env.example

```
@@ -0,0 +1,14 @@
+# Database
+MONGO_URI=mongodb://localhost:27017/github_tracker
+
+# Server Configuration
+PORT=5000
+NODE_ENV=development
+CLIENT_URL=http://localhost:5173
+COOKIE_DOMAIN=localhost
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The COOKIE_DOMAIN key should go before the NODE_ENV key

(UnorderedKey)